MdePkg/Include/Pi/PiStatusCode.h

```diff
@@ -807,6 +807,7 @@ typedef struct {
 #define EFI_SW_DXE_BS_PC_ATTEMPT_BOOT_ORDER_EVENT      (EFI_SUBCLASS_SPECIFIC | 0x00000007)
 #define EFI_SW_DXE_BS_PC_CONFIG_RESET                  (EFI_SUBCLASS_SPECIFIC | 0x00000008)
 #define EFI_SW_DXE_BS_PC_CSM_INIT                      (EFI_SUBCLASS_SPECIFIC | 0x00000009)
+#define EFI_SW_DXE_BS_PC_BOOT_OPTION_COMPLETE          (EFI_SUBCLASS_SPECIFIC | 0x0000000A)

```

Comment:
Please add the release/section of the PI Spec this status code was added to the commit message